go/vt/vtgate/plan_execute.go

```diff
@@ -68,9 +67,6 @@ func (e *Executor) newExecute(
 		safeSession,
 		logStats,
 	)
-	if err == planbuilder.ErrPlanNotSupported {
-		return err
-	}
```

Comment:
Why are we continuing to execute even if there is an error?

Reply:
we never did. we just want to log the time when the planning of the query is finished.